app/src/main/java/com/togetherjava/tjplays/games/game2048/Renderer2048.java

```diff
@@ -18,10 +18,12 @@ public final class Renderer2048 {
     private static final int TILE_SIDE = 200;
     private static final int PADDING = 20;
     private static Map<Integer, Color> tileColorMap = new HashMap<>();
+    private static int fontSize = 100; //set the font size here
```

Comment:
u just made the font size smaller in general, that would work but not how we want it to. u should calculate the fontsize depending on block value

Reply:
kk I'll see what I can do, tyvm for the reply